internal/config/channel.go

```diff
@@ -27,8 +28,9 @@ func (r *RuntimeConfig) fetchChannels(ctx context.Context, tx *sqlx.Tx) error {
 		)
 		if channelsById[c.ID] != nil {
 			channelLogger.Warnw("ignoring duplicate config for channel type")
+		} else if matched, _ := regexp.MatchString("^[a-zA-Z0-9]*$", c.Type); !matched {
```

Comment:
A trivial change to the regex would make it a bit more useful: `+` instead of `*` as an empty type and thereby empty file name doesn't make much sense (but wouldn't cause a big problem either).